src/RollbackManager.sol

```diff
@@ -17,6 +17,7 @@ import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
 ///      - Guardian queues them for execution within a specified window
 ///      - Guardian later executes or cancels the queued rollback.
 ///      - On queuing / cancelling / executing, the transactions are sent to TARGET_TIMELOCK.
+/// @custom:security-contact engineering@scopelift.co
```

Comment:
I actually don't think we should be the security contact here. In fact, I'm not sure it makes sense to have a security contact in contracts that are meant to be used as a library for lots of folks. It might be worth asking OZ about how they view this tag and it's purpose.